### ISSUE TITLE
File size is not defined

### ISSUE THREAD

**Issue by maheshsetti** [badunk/multer-s3#106]
Im trying get file size, but size key is missing. I can see only following information.
> {  
    "fieldname":"files",
    "originalname":"Art-Factory-Radha-Krishna-Painting-SDL886121492-1-1274f.jpg",
    "encoding":"7bit",
    "mimetype":"image/jpeg"
}

**Comment by rakshith-ravi** [comment]
@maheshsetti we're you ever able to find the solution to this?

**Comment by maheshsetti** [comment]
It's available in the request object.

> {
>     name: req.file.originalname,
>     size: req.file.size,
>     mime: req.file.mimetype,
>     path: req.file.key
> }

**Comment by rakshith-ravi** [comment]
Yes indeed, but in my case, however, the `req.file.size` is different than the actual size of the file uploaded to S3. Could the file size information be wrong by any chance?

**Comment by maheshsetti** [comment]
Can you share your code? It seems fine in my case.
